packages/web/docs/src/components/landing-page.tsx

```diff
@@ -29,9 +29,13 @@ const useIsomorphicLayoutEffect = typeof window !== 'undefined' ? useLayoutEffec
 
 export function IndexPage(): ReactElement {
   useIsomorphicLayoutEffect(() => {
-    // TODO: Accept a className for sidebar in the theme config?
-    const sidebarContainer = document.querySelector('.nextra-sidebar-container');
-    sidebarContainer?.parentElement?.classList.add('light');
+    // We add .light class to body to style the Headless UI
+    // portal containing search results.
+    document.body.classList.add('light');
+
+    return () => {
+      document.body.classList.remove('light');
+    };
```

Comment:
I don't get it, it's a regular practice to add `dark` class on `<html>`, why we need `light` class?

Reply:
Okay, so we have both `.light` and `.dark` classes on `html`, controlled by Tailwind, right?
And they tell our styles if they're in dark mode or light mode context, usually by affecting the `dark:` prefix.

I switched Tailwind's dark mode variant to compile to `&:not(.light *)` (from `is(.dark *)`), because
1. We have an use case where we have `.light` sectors under `html.dark` page — as the landing page is light mode only by design and with some reasoning behind it (not my decision, I'm just a soldier here, I like dark mode like every other developer).
1. I don't want to mess with Tailwind's control over `dark`/`light` class on `<html>` element.

You might notice they even have this exact code as an example in the docs.

https://tailwindcss.com/docs/dark-mode#overriding-the-dark-variant

Thus, everything is dark mode, unless it's under something with a `.light` class.

Here's the original commit from September 2, where it comes from: https://github.com/kamilkisiela/graphql-hive/commit/c7add31c36acf26c01d70583b92605f912fd9845